### DISCUSSION TITLE
How to use alternative exposure controls for divers

### DISCUSSION THREAD

**Discussion by ApneaP** [gopro/labs#972] [Q&A]
Hello
I did find this on the Labs website:
Alternative Exposure Controls.

EV adjustments. While you can just see an EV value you can now use x++ to increase from the current EV and x- - to decrease. This might be useful for dive application, as you can’t change EV while under water
White Balance adjustments, with w++ and w- - will increase or decrease white balance. Again for dive users.

I could not figure out how to use this..., can anyone help please

**Comment by dnewman-gpsw** [comment]
Create a QR Code for x++ and another for x--.  Use https://gopro.github.io/labs/control/custom/ and the Additional Commands field to type in the command. That is all you need.
